testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/siteparam.in

```diff
@@ -1,75 +1,99 @@
+		# shortens/extends high rate.
+.1		# rate slope: lower value (eg .01) straightens S shape meaning more gradual
+		# reduction effect; higher value (.5) makes abrupt transition
+.25		# inflection point (y-value of inflection point)
+0.5		# range: diff btw upper and lower rates at the limits
+
+#---- Transpiration
+# These control the tangent function (tanfunc) which affects the amount of soil
+# water extractable by transpiration.
+45.		# rate shift
+.1		# rate shape
+.5		# inflection point
+1.1		# range
+
+#---- Surface and soil temperature
 # from Parton 1978, ch. 2.2.2 Temperature-profile Submodel
-300.			# biomass limiter, 300 g/m^2 in Parton's equation for T1(avg daily temperature at the top of the soil)
-15.			# constant for T1 equation (used if biomass <= biomass limiter), 15 in Parton's equation
--4.			# constant for T1 equation (used if biomass > biomass limiter), -4 in Parton's equation
-600.        # constant for T1 equation (used if biomass > biomass limiter), 600 in Parton's equation
+300.		# biomass limiter, 300 g/m^2 in Parton's equation for T1(avg daily temperature at the top of the soil)
+15.		# constant for T1 equation (used if biomass <= biomass limiter), 15 in Parton's equation
+-4.		# constant for T1 equation (used if biomass > biomass limiter), -4 in Parton's equation
+600.		# constant for T1 equation (used if biomass > biomass limiter), 600 in Parton's equation
 0.00070		# constant for cs (soil-thermal conductivity) equation, 0.00070 in Parton's equation
-0.00030     # constant for cs equation, 0.00030 in Parton's equation
+0.00030		# constant for cs equation, 0.00030 in Parton's equation
 0.18		# constant for sh (specific heat capacity) equation, 0.18 in Parton's equation
-6.69		# constant mean air temperature (the soil temperature at the lower boundary, 180 cm) in celsius
-15.			# deltaX parameter for soil_temperature function, default is 15.  (distance between profile points in cm)  max depth (the next number) should be evenly divisible by this number
-990.		# max depth for the soil_temperature function equation, default is 180.  this number should be evenly divisible by deltaX
-0			# flag, 1 to calculate soil_temperature, 0 to not calculate soil_temperature
+4.15		# constant soil temperature (Celsius) at the lower boundary (max depth); approximate by mean annual air temperature of site
+15.		# deltaX parameter for soil_temperature function, default is 15.  (distance between profile points in cm)  max depth (the next number) should be evenly divisible by this number
+990.		# max depth for the soil_temperature function equation, default is 990.  this number should be evenly divisible by deltaX
+1		# flag, 1 to calculate soil_temperature, 0 to not calculate soil_temperature
+1       # Method for average surface temperature:
+        #     0, default prior to v8.1.0 (Parton 1978)
+        #     1, default since v8.1.0 (Parton 1984)
 
 # ---- CO2 Settings ----
-# Use biomass multiplier
```

Comment:
Same for CO2 Settings -- we should not change any of these values unless that's the task